docs/backends.md

```diff
@@ -281,6 +281,37 @@ stringData:
 type: Opaque
 ```
 
+###### Secret in a different account
+
+The usage can be done using the 'friendly' name of the secret or the arn if the secret is located in another account:
+
+**Secret in the same account**
```

Comment:
I think `###### Secret in the same account` instead of `**Secret in the same account**` for this and `**Secret in a different account**` might organize a little better

Reply:
Refactored a little bit. Sorry, most likely the md viewer that I'm using broke the formatting